packages/analytics/src/types.ts

```diff
@@ -20,7 +20,7 @@ export type Event = {
     type: string;
     timestamp?: string;
     payload?: {
-        [key: string]: string | string[] | number | number[] | boolean | null;
+        [key: string]: string | string[] | number | number[] | boolean | object | null;
```

Comment:
Are we sure that `object` is ok as payload? Would be nice to double check with data team (cc @kolomaznikt) that they can work with it.

Reply:
I already texted him on Slack. I will let you know what is the outcome of the discussion ;).